ga/src/main/java/nl/tudelft/serg/evosql/fixture/type/DBTypeSelector.java

```diff
@@ -43,21 +43,21 @@ public DBType create(int dataType, int length) {
             case Types.LONGNVARCHAR:
                 return new DBString(EvoSQLConfiguration.MAX_STRING_LENGTH, "LONGNVARCHAR");
             case Types.CHAR:
-                return new DBString(length, "CHAR");
+                return new DBString(length, "CHAR(" + length + ")");
```

Comment:
Is this a good change? If so, we could keep it I guess.

Reply:
I'm not sure what the effect will be, I guess it was originally assumed that a `CHAR` type always has a length of one, however some DB's have a syntax that support a `CHAR` with a length field. If it has no effect on the original tests, I assume we can keep this change.

Reply:
Yeah most DBs support a length on CHAR as it's usually a string, I'm fine with this.